recipe/meta.yaml

```
@@ -50,15 +52,18 @@ build:
   ignore_run_exports:
     - python *                               # [megabuild]
     - numpy *                                # [megabuild]
-  skip: true  # [win]
+
+  # TODO(baszalmstra): Remove this before merging
+  skip: true  # [not win or not (cuda_compiler_version == "12.0" or cuda_compiler == "None")]
```

```suggestion
  skip: true  # [not win or not (cuda_compiler_version == "12.0" or cuda_compiler == "None")]
  skip: true  # [py!=312]
```

just to save some resources on future builds. definitely let this one complete.